docs/release-notes/5.7.44-55.md

```diff
@@ -0,0 +1,37 @@
+# Percona Server for MySQL 5.7.44-55 (2025-08-20)
+
+--8<--- "eol.release.md"
+
+## Release Information  
+
+Percona Server for MySQL 5.7.44-55 builds on MySQL 5.7.44 Community Edition,  
+adding enterprise-grade features and security enhancements from Percona.  
+
+## Security fixes
+
+The following is the final list of fixes which will be ported to 5.7
+
+[`Bug#37117875`](https://github.com/mysql/mysql-server/commit/9d4ead06951): When the delimiter is changed from the default and a SQL statement includes an unquoted semicolon inside a MySQL-specific comment, the statement executes successfully on the source database. However, this operation causes an incomplete entry in the binlog, which results in a syntax error on the replica database.
+
+[`Bug#37829550`](https://github.com/mysql/mysql-server/commit/21a91a2b977): Enhanced `WAIT_UNTIL_SQL_THREAD_AFTER_GTIDS` validations
+
+[`Backport of WL#16949 to 8.0.`](https://github.com/mysql/mysql-server/commit/04c191b4800): A new `--commands` option has been added to the mysql client. This option, which is enabled by default, allows users to disable client-side commands when running in non-interactive mode.
+
+[`Bug#31360522`](https://github.com/mysql/mysql-server/commit/c51237cec9f): An unexpected server exit may occur when performing range queries on a table that includes a prefixed `BLOB` or `TEXT` column within its index.
+
```

Comment:
Please add the following security fix to the notes as well 
Bug#37478594(https://github.com/mysql/mysql-server/commit/fc4b73f4002)  Virtual column value may be incorrectly set to NULL on cascade update which could lead to a server crash